### PR TITLE
Aml 1907 display transfer allowance

### PR DESCRIPTION
**Pull request by robinoconnor**
This work is around the displaying and calculating of the transfer allowance in the transfer dashboard. Currently it is not in scope to work out how payments and transfers will reduce this value so it just works out the initial figure at the moment

### CODE REVIEW

File: src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAccountTransferAllowance.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+				 @financialYearEndDate = DATEFROMPARTS(@currentYear, 4, 20)
+			END
+		ELSE
+			BEGIN
+				SELECT @financialYearStartDate = DATEFROMPARTS(@currentYear - 2, 4, 20),
+				 @financialYearEndDate = DATEFROMPARTS(@currentYear - 1, 4, 20)
+			END
+	END
+
+	SELECT SUM(Amount) * 0.1 FROM [employer_financial].[TransactionLine]
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAccountTransferAllowance.sql]: The allowance %age shouldn't be hard coded.

File: src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAccountTransferAllowance.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
+﻿CREATE PROCEDURE [employer_financial].[GetAccountTransferAllowance]
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAccountTransferAllowance.sql]: The transfer allowance needs to be calculated once a year and stored as late declarations and adjustments shouldn't be taken into consideration.

Comment (robinoconnor) [reply src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAccountTransferAllowance.sql]: But payments will affect this values so we are going to need to on demand unless we change it every time a payment transaction is added to the account?

Comment (MichaelYoung1981) [reply src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAccountTransferAllowance.sql]: Yes, following our discussion yesterday my comment was rubbish!

Comment (robinoconnor) [reply src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAccountTransferAllowance.sql]: i'm going to leave this issue as it is as we will cover this when we do the payments work. This code will not be released until then as its part of the feature. How we manage payments will be the main affect of how we cache/store pre calculated values.

**Comment by robinoconnor** [comment]
I've sorted out the one issue from the code review so this should be good to go.

**Pull request merged**